training/cloud/Makefile

```
@@ -0,0 +1,33 @@
+CLOUD ?=
+HARDWARE ?= nvidia
+REGISTRY ?= quay.io
+REGISTRY_ORG ?= ai-lab
+IMAGE_NAME ?= ${HARDWARE}-bootc
+IMAGE_TAG ?= ${CLOUD}-latest
```

I think the _purpose_ of the image should be in the name. Adding the cloud provider shows it derives from the hardware specific image.

```suggestion
IMAGE_NAME ?= ${HARDWARE}-bootc-${CLOUD}
IMAGE_TAG ?= latest
```